### ISSUE TITLE
Make `orderBy` parameter actually do something

### ISSUE THREAD

**Issue by caitp** [caitp/ui-comments#2]
Okay, there are problems with this, I'm not **totally** sure how this is going to work, but there must be a way to do this sensibly, hmm? I'll look into this in a bit, after updating the demo site to take advantage of the new functionality.

**Comment by phuyem** [comment]
I don't think we need orderby. My comments have orderby button which invokes server to reload data for the new order
